g2p/tests/test_utils.py

```diff
@@ -221,6 +222,14 @@ def test_bad_normalization(self):
     def test_normalize_to_NFD_with_indices(self):
         # Usefull site to get combining character code points:
         # http://www.alanwood.net/unicode/combining_diacritical_marks.html
+        yid_multi_diacritic = "'שָׂ'"
+        print(list(yid_multi_diacritic))
```

Comment:
I would prefer you either remove the print statement, or include a message indicating what the reason is you're printing when you print, because otherwise it's a bit confusing.

Reply:
Yeah, oops, I definitely did not intend to leave these print statements around.